package NCommonServerProto;

message TInstantProcessorData {
    optional uint64 LastExportTS = 1[default = 0];
}

message THistoryProcessorData {
    optional uint64 LastEventId = 1[default = 0];
    optional string LastError = 2;
}

message TRTQueueTask {
    optional string ClassName = 1;
    optional string Data = 2;
}

message TSnapshotsDiffState {
    optional uint64 LastSnapshotId = 1[default = 0];
}

message TSnapshotsDiffTask {
    optional string SnapshotsGroupId = 1;
}